RoutingTests/OSRoutingServiceTests.swift

```diff
@@ -57,8 +58,9 @@ class OSRoutingServiceTests: XCTestCase {
             self.handler = completion
         }
 
-        func routeBetween(locations locations: [CLLocationCoordinate2D], completion: (Result<Route> -> Void)) {
-
+        func routeBetween(coordinates coordinates: [CLLocationCoordinate2D], completion: (Result<Route> -> Void)) {
```

Comment:
not specifically tied to this PR _but_ just a general Q.

When tapping points from user input, it is highly unlikely that we will be touching a point on the underlying routing graph.

So I guess my question atm is are we just providing points that are on the graph or is there some logic to connect where folk touch to the graph.

Possibly irrelevant for this initial code...but the underlying graph may also change with time...so if folk ever connected old routes with new the same problem would stand. #thoughts